software/base/src/main/java/brooklyn/entity/basic/EmptySoftwareProcessSshDriver.java

```diff
@@ -43,25 +44,22 @@ public void install() { }
     public void customize() { }
 
     @Override
-    public void setup() { }
-
-    @Override
-    public void copyInstallResources() { }
-
-    @Override
-    public void copyRuntimeResources() { }
-
-    @Override
-    public void runPreInstallCommand(String command) { }
-
-    @Override
-    public void runPostInstallCommand(String command) { }
-
-    @Override
-    public void runPreLaunchCommand(String command) { }
+    public void copyInstallResources() { 
+        Map<String, String> installFiles = entity.getConfig(SoftwareProcess.INSTALL_FILES);
+        Map<String, String> installTemplates = entity.getConfig(SoftwareProcess.INSTALL_TEMPLATES);
+        if ((installFiles!=null && !installFiles.isEmpty()) || (installTemplates!=null && !installTemplates.isEmpty())) {
```

Comment:
What's the point of guarding the `super` call? To prevent mkdir?
Whatever the reason worth a comment.